android/build.gradle.kts

```diff
@@ -1,132 +1,107 @@
-        applicationId = "com.game.quillyjumper"
-        minSdkVersion(14)
+        applicationId = "com.game.quillyjumper.android"
+        minSdkVersion(23)
         targetSdkVersion(29)
-        versionCode = 1
-        versionName = "1.0"
+        version = Versions.version
+        versionCode = Versions.androidVersionCode
     }
+
     buildTypes {
-        getByName("release") {
-            isMinifyEnabled = false
+        getByName("release").apply {
+            isMinifyEnabled = true
             proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
         }
     }
 }
-
-
-// called every time gradle gets executed, takes the native dependencies of
-// the natives configuration, and extracts them to the proper libs/ folders
-// so they get packed with the APK.
-task("copyAndroidNatives") {
-    doFirst {
-        file("libs/armeabi/").mkdirs()
-        file("libs/armeabi-v7a/").mkdirs()
-        file("libs/arm64-v8a/").mkdirs()
-        file("libs/x86_64/").mkdirs()
-        file("libs/x86/").mkdirs()
-
-        natives.files.forEach { jar ->
-            var outputDir: File? = null
-            if (jar.name.endsWith("natives-arm64-v8a.jar")) outputDir = file("libs/arm64-v8a")
-            if (jar.name.endsWith("natives-armeabi-v7a.jar")) outputDir = file("libs/armeabi-v7a")
-            if (jar.name.endsWith("natives-armeabi.jar")) outputDir = file("libs/armeabi")
-            if (jar.name.endsWith("natives-x86_64.jar")) outputDir = file("libs/x86_64")
-            if (jar.name.endsWith("natives-x86.jar")) outputDir = file("libs/x86")
-            if (outputDir != null) {
-                copy {
-                    from(zipTree(jar))
-                    into(outputDir) {
+tasks {
+    copyAndroidNatives {
+        doFirst {
+            file("libs/armeabi/").mkdirs()
+            file("libs/armeabi-v7a/").mkdirs()
+            file("libs/arm64-v8a/").mkdirs()
+            file("libs/x86_64/").mkdirs()
+            file("libs/x86/").mkdirs()
+            natives.files.forEach { jar ->
+                val outputDir = when {
+                    jar.name.endsWith("natives-arm64-v8a.jar") -> file("libs/arm64-v8a")
+                    jar.name.endsWith("natives-armeabi-v7a.jar") -> file("libs/armeabi-v7a")
+                    jar.name.endsWith("natives-armeabi.jar") -> file("libs/armeabi")
+                    jar.name.endsWith("natives-x86_64.jar") -> file("libs/x86_64")
+                    jar.name.endsWith("natives-x86.jar") -> file("libs/x86")
+                    else -> null
+                }
+                if (outputDir != null) {
+                    copy {
+                        from(zipTree(jar))
+                        into(outputDir)
                         include("*.so")
                     }
                 }
             }
         }
-    }
-}
 
-tasks.whenTaskAdded {
-    if (name.contains("package")) {
-        dependsOn("copyAndroidNatives")
     }
-}
 
-task("run") {
```

Comment:
What happened to the run task? Is it not needed?

Reply:
You would only need that when using notepad++ and who TF uses that?
That's what an IDE is for just click the run button in there instead